docs/_data/table/user-configuration.csv

```diff
@@ -1,37 +1,63 @@
+alluxio.user.block.remote.writer.class,alluxio.client.netty.NettyRemoteBlockWriter
 alluxio.user.block.size.bytes.default,512MB
+alluxio.user.block.worker.client.pool.gc.threshold.ms,300000
+alluxio.user.block.worker.client.pool.size.max,128
+alluxio.user.block.worker.client.threads,10
+alluxio.user.date.format.pattern,MM-dd-yyyy HH:mm:ss:SSS
 alluxio.user.failed.space.request.limits,3
 alluxio.user.file.buffer.bytes,8MB
 alluxio.user.file.cache.partially.read.block,true
 alluxio.user.file.copyfromlocal.write.location.policy.class,alluxio.client.file.policy.RoundRobinPolicy
 alluxio.user.file.delete.unchecked,false
 alluxio.user.file.master.client.threads,10
 alluxio.user.file.metadata.load.type,Once
-alluxio.user.file.waitcompleted.poll.ms,1000
+alluxio.user.file.passive.cache.enabled,true
+alluxio.user.file.readtype.default,CACHE_PROMOTE
+alluxio.user.file.seek.buffer.size.bytes,1MB
+alluxio.user.file.waitcompleted.poll.ms,1sec
+alluxio.user.file.worker.client.pool.gc.threshold.ms,300000
+alluxio.user.file.worker.client.pool.size.max,128
 alluxio.user.file.worker.client.threads,10
-alluxio.user.file.write.location.policy.class,alluxio.client.file.policy.LocalFirstPolicy
 alluxio.user.file.write.avoid.eviction.policy.reserved.size.bytes,0MB
-alluxio.user.file.readtype.default,CACHE_PROMOTE
-alluxio.user.file.writetype.default,MUST_CACHE
+alluxio.user.file.write.location.policy.class,alluxio.client.file.policy.LocalFirstPolicy
 alluxio.user.file.write.tier.default,0
-alluxio.user.heartbeat.interval.ms,1000
-alluxio.user.hostname,determined by localhost lookup
+alluxio.user.file.writetype.default,MUST_CACHE
+alluxio.user.heartbeat.interval.ms,1sec
+alluxio.user.hostname,
 alluxio.user.lineage.enabled,false
 alluxio.user.lineage.master.client.threads,10
+alluxio.user.local.reader.packet.size.bytes,8MB
+alluxio.user.local.writer.packet.size.bytes,64KB
+alluxio.user.network.netty.channel,
+alluxio.user.network.netty.channel.pool.disabled,false
+alluxio.user.network.netty.channel.pool.gc.threshold.ms,300000
+alluxio.user.network.netty.channel.pool.size.max,1024
+alluxio.user.network.netty.reader.buffer.size.packets,16
+alluxio.user.network.netty.reader.cancel.enabled,true
+alluxio.user.network.netty.reader.packet.size.bytes,64KB
 alluxio.user.network.netty.timeout.ms,30000
-alluxio.user.network.netty.writer.close.timeout.ms,300000
 alluxio.user.network.netty.worker.threads,0
-alluxio.user.ufs.delegation.read.buffer.size.bytes,8MB
-alluxio.user.ufs.delegation.write.buffer.size.bytes,2MB
-alluxio.user.ufs.file.reader.class,alluxio.client.netty.&#8203;NettyUnderFileSystemFileReader
-alluxio.user.ufs.file.writer.class,alluxio.client.netty.&#8203;NettyUnderFileSystemFileWriter
+alluxio.user.network.netty.writer.buffer.size.packets,16
+alluxio.user.network.netty.writer.close.timeout.ms,300000
+alluxio.user.network.netty.writer.packet.size.bytes,64KB
 alluxio.user.rpc.retry.base.sleep.ms,50
+<<<<<<< HEAD
```

Comment:
@nika11 This is unresolved merge conflict.
you should remove line 48, 49 and 51 and commit again